hc/api/tests/test_admin.py

```diff
@@ -6,14 +6,25 @@ class ApiAdminTestCase(BaseTestCase):
 
     def setUp(self):
         super(ApiAdminTestCase, self).setUp()
+        self.pwd = 'password'
+
+        # added alice as superuser and as staff.
+        self.alice.is_superuser = True
+        self.alice.is_staff = True
+        self.alice.save()
         self.check = Check.objects.create(user=self.alice, tags="foo bar")
 
-        ### Set Alice to be staff and superuser and save her :)
+        # Set Alice to be staff and superuser and save her :)
 
     def test_it_shows_channel_list_with_pushbullet(self):
-        self.client.login(username="alice@example.org", password="password")
+        self.client.login(username='alice@example.com', password="password")
```

Comment:
You should the username set in https://github.com/andela/healthchecks_akira/blob/3692db7cc498ad6bb73461580fcae821edae24e4/hc/test.py#L13